### ISSUE TITLE
Switching teams should be called desertion

### ISSUE THREAD

**Issue by Dany0** [iamgreaser/iceball#197]
Maybe it will dissuade people from changing teams when theirs is being wrecked.

**Comment by rakiru** [comment]
The people that change team to be on the winning side are the people least likely to care about being called deserters. It could be a more fun way to word it though (or leaving the server entirely).

**Comment by RSTFS** [comment]
Or desertion is for when you leave, and betrayal is when you switch.

**Comment by rakiru** [comment]
@RSTFS Yes, that's along the lines of what I was thinking. I couldn't think of the word though. That would be my preferred way to do this, if any.

**Comment by Dany0** [comment]
>Do you wish to desert your comrades of ${server_name}?

>The spymaster offers you to betray your team and switch to team, do you accept?

**Comment by rakiru** [comment]
"Spymaster"?

Sounds good though. Reminds me of the quit message of Quake and stuff.

**Comment by iamgreaser** [comment]
With respect to team switching, usually when I team switch it's to fix balance.

**Comment by epicfacethe3rd** [comment]
yeah, GM has a good point. maybe a system that just says "balanced" for autobalances and "deserter/desertion" for plain old team switching.

**Comment by rakiru** [comment]
There is no auto-balance though, and even if there was, it wouldn't be perfect all the time. I still think it adds a nice touch though. Even if not in the team switch menu, the chat message or something could be cool.

**Comment by bigtoilet** [comment]
I agree with RSTFS and rakiru. Desertion is not switching, its leaving.
Message when exiting game: Desertion is an act of treason, do you wish to continue?
Message when switching teams: Fighting for the enemy is an act of treason, do you wish to continue?

Punishment for these and other acts of treason or misconduct (team killing, chat spam) would be interesting. Perhaps if you switch from one team to another you are no longer welcome at the former team if you decide to switch back?

**Comment by rakiru** [comment]
I don't think this should have any actual effect on the game. If it is added, it should be purely a fun way of wording the existing messages.

**Comment by bigtoilet** [comment]
Yeah, it certainly doesn't need to be anything more than wording. I only mentioned punishment as an idea that I might implement somehow if I had a server myself.

**Comment by epicfacethe3rd** [comment]
an autobalance feature would be useful. perhaps a voting system? like, vote for votekick (${user} was sent to hell.) afk kicking, (${user} died of a coma.) balancing, (${user} was balanced against his own will) and forced team switching. (${user} got stuck in a ditch for ${time}).

**Comment by rakiru** [comment]
Auto-balance and voting are separate issues.

**Comment by neppermint** [comment]
>The people that change team to be on the winning side are the people least likely to care about being called deserters

As someone who changes teams in every game, I'd rather be mocked and called a deserter than be stuck on a worthless team getting destroyed.

**Comment by 10se1ucgo** [comment]
Well, then you're an asshole :D
